### PR TITLE
revise config to allow for self-healing, some fixes to graphs

### PR DESCRIPTION
**Pull request by Rahien**
## Description
revise config to allow for self-healing, some fixes to graphs

also fix some graphs not marked as bestuurseenheid graphs, fix had non-bestuurseenheid graphs with fraction content, which confused ldes healing

Requires the work in https://github.com/redpencilio/ldes-delta-pusher-service/pull/9 to be activated

## How to test

see the linked pr 
## Links to other PR's

- https://github.com/redpencilio/ldes-delta-pusher-service/pull/9

### CODE REVIEW

File: config/ldes-delta-pusher/handle-decision-type.ts [hunk 1]
```diff
@@ -1,27 +1,34 @@
-import { query } from "mu";
```
Comment (Rahien) [inline config/ldes-delta-pusher/handle-decision-type.ts:1]: use sudo queries as we need to trigger dispatch from the ldes-delta-pusher's cron for healing so we don't have a session to use for regular query.

Comment (Rahien) [reply config/ldes-delta-pusher/handle-decision-type.ts:1]: this happens in other files too

File: config/ldes-delta-pusher/handle-regular-types.ts [hunk 2]
```diff
@@ -1,28 +1,31 @@
 import { Changeset } from "../types";
-import { query } from "mu";
+import { querySudo } from "@lblod/mu-auth-sudo";
 import { publishInterestingSubjects } from "./handle-types-util";
 import { InterestingSubject, LDES_TYPE, TypesWithFilter } from "./publisher";
+import { ldesInstances } from "./ldes-instances";
 
 const regularTypesToLDESMapping: {
   [key: string]: LDES_TYPE | TypesWithFilter;
-} = {
```
Comment (Rahien) [inline config/ldes-delta-pusher/handle-regular-types.ts:8]: all of these configs were merged into ldes-instances.ts so it's in one place

File: config/ldes-delta-pusher/initialization.ts [hunk 3]
```diff
@@ -20,59 +13,22 @@ const extraWhere = `
   }
 `;
 
-export const initialization = {
```
Comment (Rahien) [inline config/ldes-delta-pusher/initialization.ts:23]: again: merged into ldes-instances and transformed here to keep the format

File: config/ldes-delta-pusher/ldes-instances.ts [hunk 4]
```diff
@@ -0,0 +1,232 @@
+export const ldesInstances = {
+  public: {
```
Comment (Rahien) [inline config/ldes-delta-pusher/ldes-instances.ts:2]: config in one place only!

Comment (Rahien) [reply config/ldes-delta-pusher/ldes-instances.ts:2]: well except for special cases mandataris and besluit, that would have taken it a little too far

File: config/ldes-delta-pusher/publisher.ts [hunk 5]
```diff
@@ -85,94 +85,6 @@ const datatypeNames = {
   "http://www.w3.org/2001/XMLSchema#boolean": "bool",
 };
 
-const defaultProperties = [
```
Comment (Rahien) [inline config/ldes-delta-pusher/publisher.ts:88]: both extracted to ldes-instances too

File: config/virtuoso/virtuoso-production.ini [hunk 6]
```diff
@@ -95,11 +95,11 @@ TransactionAfterImageLimit = 150000000	; max 150MB transaction, total transient
 ;NumberOfBuffers          = 340000
 ;MaxDirtyBuffers          = 250000
 ;; Uncomment next two lines if there is 8 GB system memory free
-; NumberOfBuffers          = 680000
```
Comment (Rahien) [inline config/virtuoso/virtuoso-production.ini:98]: servers are not that big, changing this to give them a little more breathing room (our data isn't big either so should be fine)

File: docker-compose.dev.yml [hunk 7]
```diff
@@ -93,6 +93,9 @@ services:
       BASE_URL: "http://localhost/streams/ldes/"
     ports:
       - "6666:80"
```
Comment (Rahien) [inline docker-compose.dev.yml:95]: want to be able to run ldes-delta-pusher as a debug compose on the side if we want to

**Review by JonasVanHoof — APPROVED** [review]
🚀

**Pull request merged**